### PR TITLE
The final UI version

### PR DESCRIPTION
**Pull request by arushigarg18**

### CODE REVIEW

File: ui/ui_vfinal/scripts/process_query.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ * @return {string} queryResult.status - Indicates whether gcp function returned results successfully
+ * @return {string} queryResult.jsonQuery - The json for the submitted query
+ * @return {Array} queryResult.outputTable - The output table of the query submitted 
+ * @return {Object[]} queryResult.suggestions - The list of suggestions related to the submitted query
+ * @return {?Array} queryResult.filtersPassed - The list of true-false values to insert output as column on filtering
+ * @return {?Array} queryResult.inTopK -  The list of true-false values to insert output as column in topk
+ */
+function evalQuery(
+    intent, 
+    metric,
```
Comment (SowmithManepalli) [inline ui/ui_vfinal/scripts/process_query.js:57]: if i understand it right , this is just intent name and aspects, if an intent doesnt have few aspects they will be null right ? if yes can you add this to the comment at the top.

Comment (SowmithManepalli) [reply ui/ui_vfinal/scripts/process_query.js:57]: Also , can you link what are the aspects required for each intent information in the readme.

File: ui/ui_vfinal/aspects/limit/limit_js.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!-- Javascript functions for aspect: limit -->
```
Comment (SowmithManepalli) [inline ui/ui_vfinal/aspects/limit/limit_js.html:1]: Also explain the directory structure in the readme.

File: ui/ui_vfinal/scripts/process_query.js (hunk 1 above)
Comment (arushigarg18) [reply ui/ui_vfinal/scripts/process_query.js:57]: I am using this notation: {?Array} to indicate null or array
Resource: https://jsdoc.app/tags-type.html
A number or null.
{?number} | This indicates that the type is either the specified type, or null.
-- | --

Comment (SowmithManepalli) [reply ui/ui_vfinal/scripts/process_query.js:57]: Yeah cool sounds good.

**Pull request merged**